content/news/2021.pml

```diff
@@ -612,7 +612,9 @@ Older News: <%=reflist%>
 <!----------------------------------------------------------->
 
 <div class="news-item">
-  23 Jun 2021: Horizon Robotics joins the seL4 Foundation
+  <a name="horizon">
```

Comment:
Better use an ID here that is unique, there might be another news item about them one day.
```suggestion
  <a name="horizon-20210623">
```